server/channels/app/support_packet.go

```diff
@@ -82,26 +84,30 @@ func (a *App) GenerateSupportPacket(c request.CTX, options *model.SupportPacketO
 
 	wg.Wait()
 
-	if pluginsEnvironment := a.GetPluginsEnvironment(); pluginsEnvironment != nil {
-		pluginContext := pluginContext(c)
-		for _, id := range options.PluginPackets {
-			hooks, err := pluginsEnvironment.HooksForPlugin(id)
-			if err != nil {
-				c.Logger().Error("Failed to call hooks for plugin", mlog.Err(err), mlog.String("plugin", id))
-				warnings = multierror.Append(warnings, err)
-				continue
-			}
-			pluginData, err := hooks.GenerateSupportData(pluginContext)
-			if err != nil {
-				c.Logger().Warn("Failed to generate plugin file for Support Packet", mlog.Err(err), mlog.String("plugin", id))
-				warnings = multierror.Append(warnings, err)
-				continue
-			}
-			for _, data := range pluginData {
-				fileDatas = append(fileDatas, *data)
+	pluginContext := pluginContext(c)
+	a.ch.RunMultiHook(func(hooks plugin.Hooks, manifest *model.Manifest) bool {
+		// If the plugin defined the support_packet prop it means there is a UI element to include it in the support packet.
+		// Check if the plugin is in the list of plugins to include in the Support Packet.
+		if _, ok := manifest.Props["support_packet"]; ok {
+			if !slices.Contains(options.PluginPackets, manifest.Id) {
+				return false
 			}
```

Comment:
Could this simplified to: 
```suggestion
			return slices.Contains(options.PluginPackets, manifest.Id)
```